src/test/java/com/google/devtools/build/lib/analysis/actions/StrippingPathMapperTest.java

```diff
@@ -171,12 +175,12 @@ public void starlarkRule_optedInViaExecutionRequirements() throws Exception {
     String outDir = analysisMock.getProductName() + "-out";
     assertThat(spawn.getArguments().stream().collect(toImmutableList()))
```

Comment:
im seeing failures for this test with the following error? Are you seeing the same?  ERROR:

```
missing (1)   : <blaze-out/cfg/bin/pkg/gen_src.txt>
unexpected (1): <pkg/gen_src.txt:blaze-out/cfg/bin/pkg/gen_src.txt:blaze-out/cfg/bin:blaze-out/cfg/bin/pkg>
---
expected      : [blaze-out/cfg/bin/tool/tool, blaze-out/cfg/bin/pkg/out.bin, -source, <blaze-out/cfg/bin/pkg/gen_src.txt>, -source, <pkg/source.txt:pkg/source.txt::pkg>]
but was       : [blaze-out/cfg/bin/tool/tool, blaze-out/cfg/bin/pkg/out.bin, -source, <pkg/gen_src.txt:blaze-out/cfg/bin/pkg/gen_src.txt:blaze-out/cfg/bin:blaze-out/cfg/bin/pkg>, -source, <pkg/source.txt:pkg/source.txt::pkg>]
```

quick observation, the unmapped commandline looks like this `blaze-out/k8-fastbuild/bin/pkg/out.bin -source <pkg/gen_src.txt:blaze-out/k8-fastbuild/bin/pkg/gen_src.txt:blaze-out/k8-fastbuild/bin:blaze-out/k8-fastbuild/bin/pkg> -source <pkg/source.txt:pkg/source.txt::pkg>` which makes sense why the mapped output looks like above.

Reply:
There was an unresolved merge conflict, I now pushed a new version with all tests passing.